<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отображение карты</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" rowspan="2">
     <widget class="QLabel" name="g_map">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="g_layer1">
        <property name="text">
         <string>схема</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="g_layer2">
        <property name="text">
         <string>автодороги</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="g_layer3">
        <property name="text">
         <string>общ. траснпорт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lightButton">
        <property name="text">
         <string>Светлая</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="darkButton">
        <property name="text">
         <string>Тёмная</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
